tests/testthat/test-0_portfolio_input_check_functions.R

```diff
@@ -0,0 +1,26 @@
+portfolio <- tibble::tribble(
```

Comment:
this while file is from somewhere else, no?

Reply:
should this file be gone entirely?

Reply:
no... I split them out into tests per function.... maybe you merged master with an old version of master?

Reply:
that wasn't clear.... yes, this file should be completely gone because it has been superseded by 4 individual test files
- test-add_fin_data.R
- test-add_fund_portfolio.R
- test-calculate_fund_portfolio.R
- test-clean_unmatched_holdings.R